app/src/main/java/org/mozilla/focus/FocusApplication.java

```diff
@@ -31,17 +31,21 @@ public void onCreate() {
     }
 
     private void enableStrictMode() {
-        if (AppConstants.isReleaseBuild()) {
+        // Android/WebView sometimes commit strict mode violations, see e.g.
+        // https://github.com/mozilla-mobile/focus-android/issues/660
+        if (AppConstants.isReleaseBuild() || AppConstants.isBetaBuild()) {
             return;
         }
 
         StrictMode.setThreadPolicy(new StrictMode.ThreadPolicy.Builder()
                 .detectAll()
+                .penaltyLog()
                 .penaltyDeath()
```

Comment:
penaltyLog() and penalyDeath() at the same time doesn't really make sense, I think.

With just logging it's easy to miss things - I think I would prefer false positive crashes (in Dev) over missing penalties because they are only logged.
At least for ThreadPolicy we could use penaltyDialog() - maybe even on Beta.

How about this:
* Thread Policy
  * Beta: Dialog
  * Debug: Dialog
* VmPolicy
  * Beta: Log
  * Debug: Death